solidus_globalize.gemspec

```diff
@@ -2,18 +2,18 @@
 lib = File.expand_path('../lib/', __FILE__)
 $LOAD_PATH.unshift lib unless $LOAD_PATH.include?(lib)
 
-require 'spree_globalize/version'
+require 'solidus_globalize/version'
 
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
-  s.name        = 'spree_globalize'
-  s.version     = SpreeGlobalize.version
+  s.name        = 'solidus_globalize'
+  s.version     = SolidusGlobalize.version
   s.summary     = 'Provides globalization features for Spreecommerce.'
   s.description = s.summary
 
   s.author      = 'Sean Schofield'
```

Comment:
@tvdeyen ah sorry, I thought you was talking about my diffs, what about me, you and @alepore at the moment? Is it ok for you?